internal/pkg/aws/elbv2/elbv2.go

```diff
@@ -54,5 +55,27 @@ func (e *ELBV2) TargetsHealth(targetGroupARN string) ([]*TargetHealth, error) {
 
 // TargetID returns the target's ID, which is either an instance or an IP address.
 func (t *TargetHealth) TargetID() string {
+	return t.targetID()
+}
+
+// HealthStatus contains the health status info of a target.
+type HealthStatus struct {
+	TargetID          string `json:"targetID"`
+	HealthDescription string `json:"healthDescription"`
+	HealthState       string `json:"healthState"`
+	HealthReason      string `json:"healthReason"`
```

Comment:
My preference would be for these to be:
```suggestion
	TargetID          string `json:"targetID"`
	HealthDescription string `json:"description"`
	HealthState       string `json:"state"`
	HealthReason      string `json:"reason"`
```
and then provide a `"health"` key in the client to wrap this information:
```
Health HealthStatus `json:"health"`
```